phpunit.xml.dist

```diff
@@ -14,6 +14,7 @@
     <testsuites>
         <testsuite name="Behat Gherkin test suite">
             <directory>./tests/Behat/Gherkin/</directory>
+            <directory>./tests/Upstream/</directory>
```

Comment:
I suggest putting the upstream test in the `./tests/Behat/Gherkin/` folder with others (and later switching to PSR-4 instead of PSR-0 to remove these 2 levels of nesting)

Reply:
Can do - they should probably have some sort of separation though (maybe `@group` is enough)

Reply:
but that's easy, as there is only one class on which the group needs to be applied.